cairo/tests/ethereum/cancun/vm/instructions/test_storage.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from hypothesis import given
+
+from ethereum.cancun.vm.instructions.storage import sload
+from tests.utils.args_gen import Evm
+from tests.utils.evm_builder import EvmBuilder
+
+pytestmark = pytest.mark.python_vm
+
+
+class TestStorage:
+    @given(evm=EvmBuilder().with_stack().with_env().with_gas_left().build())
```

Comment:
the default strategy for env is environment_lite which is doing a st.fromType(State) 
Should this be updated to use the state strategy that was refined on the precedent PRs ?

Reply:
used a local composite strategy. For the state, it doesn't matter much what's drawn from get_storage.